gpMgmt/bin/gpload_test/gpload2/query37.ans

```diff
@@ -1,4 +1,4 @@
-NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 's1' as the Cloudberry Database data distribution key for this table.
+NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 's1' as the Apache Cloudberry data distribution key for this table.
```

Comment:
I think we shouldn't  change the message from code (*.c, python, bin and etc) files, this will introduce many file diffs which are pointless.
And the worse is, any databases like our commercial ones or tools based on CBDB would fail on CI due to that and have to fix them.
Rename the DOC part is enough. `Cloudberry Database` doesn't have conflict with `Apache Cloudberry `

Reply:
> I think we shouldn't change the message from code (*.c, python, bin and etc) files, this will introduce many file diffs which are pointless. And the worse is, any databases like our commercial ones or tools based on CBDB would fail on CI due to that and have to fix them. Rename the DOC part is enough. `Cloudberry Database` doesn't have conflict with `Apache Cloudberry `

I tend to agree with @avamingli . However, there are several important user facing changes which would be appropriate to make. Like that in `configure.ac` which go into `version()`. There are also user facing help content we should consider making.

This is not an easy task but it has to be performed on some level.

Reply:
Aha, it’s genuinely challenging work.

The project was originally known as `Cloudberry Database` before we joined the ASF Incubator; it has since been rebranded as `Apache Cloudberry`. We need to update all instances of the old name in the source code, website, and other repositories for better compliance.

This task should be completed sooner rather than later, as it could become a significant blocker for our future Apache release or graduation checks.

Additionally, we are collaborating with others in the ASF community as individuals. It’s important to adopt a community-first mindset, rather than a commercial perspective, even though some contributors may be sponsored or employed by commercial companies.